generators/generator-base-private.js

```diff
@@ -625,17 +625,16 @@ module.exports = class extends Generator {
     }
 
     /**
-     * Execute callback if git is installed
+     * Checks if git is installed.
      *
-     * @param {function} callback - function to be called if git is installed
+     * @return {boolean} true if installed; false otherwise..
      */
-    isGitInstalled(callback) {
-        this.gitExec('--version', { trace: false }, code => {
-            if (code !== 0) {
-                this.warning('git is not found on your computer.\n', ` Install git: ${chalk.yellow('https://git-scm.com/')}`);
-            }
-            if (callback) callback(code);
-        });
+    isGitInstalled() {
```

Comment:
Do we want to keep callback compatibility?
It's really simple to add.

Reply:
You are right. Will add it back

Reply:
Added back